plugin/sampling/strategystore/static/strategy_store.go

```diff
@@ -41,6 +47,18 @@ func NewStrategyStore(options Options, logger *zap.Logger) (ss.StrategyStore, er
 		logger:            logger,
 		serviceStrategies: make(map[string]*sampling.SamplingStrategyResponse),
 	}
+
+	// new instance of viper to watch sampling strategies file
+	viper.AddConfigPath(options.StrategiesFile)
```

Comment:
Why do we need to depend on viper (and especially on its global state) for watching?

Note that we already have a file watcher implemented for UI config.

Reply:
Swapped for `fsnotify` watcher.